src/DIRAC/WorkloadManagementSystem/DB/JobDB.py

```diff
@@ -908,6 +908,7 @@ def insertNewJobIntoDB(
             "SubmissionTime": str(datetime.datetime.utcnow()),
             "Owner": owner,
             "OwnerGroup": ownerGroup,
+            "VO": getVOForGroup(ownerGroup),
```

Comment:
Is that enough?
Don't you have to add the `VO` field in `JobDB.sql` as well?

Reply:
Yes, you're correct.